swarm/src/connection/pool.rs

```diff
@@ -495,6 +497,10 @@ where
     ) {
         let connection = connection.extract();
 
+        let span = tracing::error_span!("peer id:", %obtained_peer_id, connection_id = tracing::field::Empty);
```

Comment:
```suggestion
        let span = tracing::error_span!("connection", %peer_id, connection_id = tracing::field::Empty);
```

I think the name of the span should be `connection`.

Plus, it would be good to record the other peer's IP address as well I'd say.